t/mojo/morbo.t

```diff
@@ -136,6 +136,49 @@ subtest 'Broken symlink' => sub {
   ok !$warned, 'no warnings';
 };
 
+subtest 'Randomized Request IDs' => sub {
+  my @ids;
+  my $src = <<'EOF';
+use Mojolicious::Lite;
+
+app->log->level('fatal');
+
+# Return "request_id"
+get '/request_id' => sub {
+  my $c = shift;
+  $c->render( text => $c->req->request_id );
+};
+
+app->start;
+EOF
+
+  # Fetch initial request_ids
+  $script->spurt($src);
+  sleep 3;
+
+  $tx = $ua->get("http://127.0.0.1:$port/request_id");
```

Comment:
You're using a new scope, why is this a global?

Reply:
~~Cargo-culting~~ — er, I mean, *"Standing on the Shoulders of Giants"* — is why I reused the global `$tx` and `$ua`, etc.  I was only using `subtest` as a means of *grouping* the tests, not necessarily because of the new lexical scope.

Would you rather I just leave them at the top level?  (Since going through all that effort again in a subtest seems wasteful... ***That's*** not the part of the system that's actually being tested.  «shrug»)

Reply:
The test has been adjusted in the meantime and should answer all questions. 😁

Reply:
Ahh!  I see what you did there, thanks!!  :-D

BTW: Not to blaspheme, or anything, but I will say that it felt wrong that the "previous" test was responsible for updating the temporary file, and having to manually remember to increment the number seems like a footgun just waiting to go off...

For instance, to maintain consistency, I had to diddle around up in the *'Update script without changing mtime'* subtest — which wasn't "mine" — adding `$started = $started->sibling('started4.txt')`.

[With faux indignation:]  "And you yelled at ***\*ME\**** for global variables?!?"  (I'm so totally «j/k», BTW!!)  ☮️

Anyway, I've tweaked it to do:

```
my $dir = tempdir;    # This is still global...

subtest 'Blah Blah Blah' => sub {
  my $started = $dir->child('started.txt');
  ok !-f $started, 'started.txt does not exist';

  # $script->spurt(...), etc.
  sleep 0.1 until -f $started;
  ok $started->remove, 'started.txt removed';

  # my $tx = $ua->get(...)
};
```

... which seems to work.  (Would that be a separate PR?  Or a second commit in this one?  Sorry for sounding like such a newb, but *squashing* it into the current commit seems like it violates the "Do One Thing and Do It Well" Single-Responsibility Principle.  But maybe that's okay, in this case?  «shrug»)

Anyway, I'm sure one could ~~waste~~ I mean, "spend" a ton of time making it more robust — e.g., generating random filenames (`File::Temp::tempnam()` ?), and using `mkdir` instead of `touch` 'cuz the former is atomic, and all that — but I'm pretty certain that's beginning to cross into the realm of *"Better Is The Enemy Of Good"* and *"You Ain't Gonna Need It"*, etc.  :-D

Thanks again!